docs/README.md

```diff
@@ -1,30 +1,34 @@
-This guide will demonstrate how to regenerate the plots used in the documentation here.
-The requirements are
-1) Have docker installed
-2) Download the [PlotData] zip file containing the scenarios used
+### Requirements
+This guide will demonstrate how to regenerate the plots used in the
+documentation here. The only requirement is that docker is installed. The
```

Comment:
Should `here` links to https://breakthrough-energy.github.io/docs/postreise/index.html ?

Reply:
Done